### ISSUE TITLE
Sticky ListView Section Header Slides Under Facebook Tab Bar

### ISSUE THREAD

**Issue by jhprks** [brentvatne/react-native-scrollable-tab-view#6]
I've been having hard time trying to get ListView's section header to stick to the bottom of Facebook TabBar, its default is to stick to the top of the screen. When I scroll, the section header slides under the tab-bar. Any thoughts on this? Anything I should change in FacebookTabBar.js to make this work?

Without tab bar at the top.
![nobar](https://cloud.githubusercontent.com/assets/1529857/8390937/a7c828bc-1c61-11e5-9940-e8d887f90ebb.gif)

With tab bar at the top. (Strange how GIF version doesn't show the full video properly, just imagine that the list is scrolled a lot and the section header slides under the tab bar).
![withbar](https://cloud.githubusercontent.com/assets/1529857/8390938/b8906042-1c61-11e5-9c78-e038a6bf1eca.gif)

**Comment by jhprks** [comment]
This question is more appropriate for stackoverflow, I'll close this since it doesn't look like it's an inherent issue with react-native-scrollable-tab-view, but I'd like to place a link here that points to this question which I posted on stackoverflow, just in case if you wish to chime in. http://stackoverflow.com/questions/31066437/how-to-get-listview-section-header-to-stick

**Comment by brentvatne** [comment]
Interesting! Thanks for the report @jhprks - did you happen to put the ListView inside of a ScrollView? Can you post the code? I have no problem with having this issue on here too!

**Comment by jhprks** [comment]
Cool! So, the ListView is in its own file called Category.js, I imported this into index.io.js file which renders ScrollableTabView/ScrollView.

```
<View style={styles.container}>
	<ScrollableTabView renderTabBar={() => <FacebookTabBar />}>
		<ScrollView tabTitle='Search' tabLabel="ion|ios-search" style={styles.tabView}>
			<Category />
		</ScrollView>
...
   </ScrollableTabView>
</View>
```

I tried a few more things and found out that wrapping `<Category>` with `<View style={styles.categoryContain}>`  and styling it in the following way got me closer to what I wanted.

```
var deviceHeight = require('Dimensions').get('window').height;

...

var styles = StyleSheet.create({
  categoryContain: {
    top: 0,
    height: deviceHeight,
  },
});
```

Okay, so far with the above code, the section header properly sticks to the bottom of the tab bar - this is certainly what I wanted to see, but it only seems to work properly when I start out with **pulling up** the list.

![good](https://cloud.githubusercontent.com/assets/1529857/8401119/d22e34ec-1dde-11e5-8bce-51e3bc321d2f.gif)

But when I start out with **pulling down** and then **pulling up** the list, the section header gets hidden behind the tab bar, and it stays hidden.

![bad](https://cloud.githubusercontent.com/assets/1529857/8401123/db909dae-1dde-11e5-8f74-aeedd7a30ed8.gif)
